### DISCUSSION TITLE
audio team only onboard

### DISCUSSION THREAD

**Discussion by hgt27** [f1multiviewer/issue-tracker#121] [Feature requests]
hello, I think it would be useful to add an audio team only for persons with a slow network or people who don't want to have allso the onboard.

i don't know if it's possible but I'm dropping this idea here, just in case!

**Comment by JJWatMyself** [comment]
Can you clarify what this means?

I think you are asking for a radio like experience and no video at all. If so then you should use the Android/iOS app called 'Formula 1'. This has a full stat interface with radio.

**Reply by hgt27** [reply]
on the app you cant select a specific radio + i use a headphone when i watch a race so i cant hear my phone when there's a new radio

**Comment by jeppeolesen** [comment]
Hey @hgt27 

We supply the streams and audio channels as made available by F1 TV - we don't have a way to extract just the audio from a stream in a way that doesn't break F1 TV Terms of Service.
